### ISSUE TITLE
Add doc-block comments to Uniqness trait

### ISSUE THREAD

**Issue by daydiff** [daydiff/yii2-unique-command#1]

**Comment by elchroy** [comment]
Hi @daydiff 
I'm interested in working on this. But I'm having some issues setting up my composer dependencies.

```
Problem 1
    - yiisoft/yii2 2.0.9 requires bower-asset/jquery 2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable -> no matching package found.
    - yiisoft/yii2 2.0.8 requires bower-asset/jquery 2.2.*@stable | 2.1.*@stable | 1.11.*@stable -> no matching package found.
    - yiisoft/yii2 2.0.7 requires bower-asset/jquery 2.2.*@stable | 2.1.*@stable | 1.11.*@stable -> no matching package found.
    - yiisoft/yii2 2.0.6 requires bower-asset/jquery 2.1.*@stable | 1.11.*@stable -> no matching package found.
    - Installation request for yiisoft/yii2 >=2.0.6@stable -> satisfiable by yiisoft/yii2[2.0.6, 2.0.7, 2.0.8, 2.0.9].
```

Do you mind helping out?

**Comment by daydiff** [comment]
Closed by #5
